tests/lang/parse-fail-uft8.err.exp

```diff
@@ -0,0 +1,6 @@
+error: syntax error, unexpected invalid token, expecting end of file
```

Comment:
This file should be named `parse-fail-utf8.err.exp`, not `parse-fail-uft8.err.exp`.